src/react-apps/applications/runtime/src/features/form/containers/withFormElementHandlers.tsx

```diff
@@ -84,15 +170,62 @@ export const formComponentWithHandlers = (WrappedComponent: React.ComponentType<
 
       return (
         <>
-          {this.renderLabel()}
-          {this.renderDescription()}
-          <WrappedComponent
-            id={id}
-            text={text}
-            handleDataChange={this.handleDataUpdate}
-            {...passThroughProps}
-          />
-          {this.errorMessage()}
+          <Grid
+            container={true}
+            direction={'column'}
+          >
+            <Grid
+              container={true}
+              direction={'row'}
+              spacing={2}
+            >
+              <Grid item={true}>
+                <Grid
+                  container={true}
+                  direction={'column'}
+                >
+                  <Grid item={true}>
+                    {this.renderLabel()}
+                  </Grid>
+                  <Grid item={true}>
+                    {this.renderDescription()}
+                  </Grid>
+                </Grid>
+              </Grid>
+            </Grid>
+            <WrappedComponent
+              id={id}
+              text={text}
+              handleDataChange={this.handleDataUpdate}
+              {...passThroughProps}
+            />
+            {this.errorMessage()}
+          </Grid>
+          {!!helpIconRef ?
```

Comment:
How about using && and skip the ternary operator and the unnecessary ": null"?

Reply:
Because the tenary operation returns false and not null. Which material complains about PropTypes.

Reply:
Thought it was inside of the popover. Changing to `!!helpIconRef && ...` without the `:null`.